app/controllers/application_controller.rb

```diff
@@ -2,4 +2,40 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  helper_method :current_robot
+  before_action :current_order
```

Comment:
Where you put any of your private code below the `private` keyword, similar to what you would do with a params methods